src/applications/static-pages/ezr-tera-alert/components/App/index.unit.spec.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import { App } from '.';
+
+describe('ezr submission options', () => {
```

Comment:
Can you update the following to actually reflect this app:

1. Update the name of this test to reflect what you're actually testing
2. Update the test IDs to reflect enabled/disabled instead of auth/guest
3. Remove the selector check for the link. Both alerts have links and your first test will fail
4. Remove the selector check for the heading. Both alerts have headings